bundle/jsky.app.ot/src/main/scala/jsky/app/ot/ags/BagsManager.scala

```diff
@@ -106,7 +108,7 @@ object BagsManager {
     }
   }
 
-  private val NumThreads = math.min(1, Runtime.getRuntime.availableProcessors())
+  private val NumThreads = math.max(1, Runtime.getRuntime.availableProcessors()-1)
```

Comment:
Sorry: are you suggesting having more than the number of available processors?
At first, we had gone with 5 fixed, but I wasn't sure how to best proceed.
Even with a single thread, things proved to be generally "speedy enough", it seemed to me, when importing and working on an observation, so I'm not sure the best way to handle this.